addons/mail/models/mail_thread.py

```diff
@@ -2291,9 +2282,10 @@ def _notify_thread_by_email(self, message, recipients_data, msg_vals=False,
             # {company, is_discussion, lang, message, model_description, record, record_name, signature, subtype, tracking_values, website_url}
             # {actions, button_access, has_button_access, recipients}
 
-            if base_template:
-                mail_body = base_template._render(render_values, engine='ir.qweb', minimal_qcontext=True)
-            else:
+            mail_body = self.env['ir.qweb']._render(template_xmlid, render_values, minimal_qcontext=True, raise_if_not_found=False, lang=template_values['lang'])
+            if not mail_body:
+                _logger.warning('QWeb template %s not found when sending notification emails. Sending without layouting.', template_xmlid)
```

Comment:
Note that this logger does not seem right to me. Either the template was not found, either it was void. But the error is not really correct.

Reply:
using the raise_if_not_found option, if the template doesn't exist then it just returns an empty string. QWeb acts as if there is nothing to render.